brands/shop/confectionery.json

```diff
@@ -39,6 +39,19 @@
       "shop": "confectionery"
     }
   },
+  "shop/confectionery|Kilwins": {
+    "countryCodes": ["us"],
+    "matchTags": ["amenity/ice_cream"],
+    "tags": {
+      "alt_name": "Kilwin's",
```

Comment:
We currently can't support having the alternate names very similar to the main names (different only in spaces, case, punctuation), so I am going to remove the `alt_name`, but other than that the PR looks ok 👍 

There haven't been any other brands where this been an issue, but I could probably change the matching code to support it if it matters a lot to people.  We don't use `alt_name` much so it hasn't been a problem yet.

Reply:
Got it.